lib/logstash/outputs/opensearch.rb

```diff
@@ -200,6 +200,13 @@ class LogStash::Outputs::OpenSearch < LogStash::Outputs::Base
   # here like `pipeline => "%{INGEST_PIPELINE}"`
   config :pipeline, :validate => :string, :default => nil
 
+  # If enabled, skip the GET request to the Healthcheck endpoint to get the server version.
+  # The HEAD request is still done to check if the server is alive. 
+  config :skip_healthcheck, :validate => :boolean, :default => false
+
+  # The OpenSearch server major version to use when it's not available from the Healthcheck endpoint.
+  config :default_server_major_version, :validate => :number, :default => 2
```

Comment:
If the user skips the healthcheck, maybe it would be good to have them explicitly set the version? Otherwise, they might end up with unexpected behavior. I could go either way on it, but it seems that this advanced feature might require an explicit declaration on their part.

Thoughts? @deepdatta , @sshivanii

Reply:
I agree, we can get rid of the skip_heathcheck config. 
Put logic in get_version(url) to check if the response body is empty, if so use the default_server_major_version, if its not set then throw an exception. Thoughts @dlvenable, @sshivanii

Reply:
@deepdatta , This sounds good. The one change I'd suggestion:
If the `get_version(url)` throws an exception (perhaps due to 404 or 403), then use `default_server_major_version`.

Again, if it is not set, we should throw an exception in that case.